react/typings/global.ts

```diff
@@ -12,4 +12,9 @@ type SlideDirectionType =
   | 'rightToLeft'
   | 'leftToRight'
 
-type MinicartVariationType = 'popup' | 'drawer' | 'link' | 'popupWithLink'
+type MinicartVariationType =
+  | 'popup'
+  | 'drawer'
+  | 'link'
+  | 'popupWithLink'
+  | 'block'
```

Comment:
What do you think to use a better name for it? For me "block", seems that the minicart is "blocked", you know?

Reply:
@igorbrasileiro the understanding is that this will work as a regular `block` occupying real space on the page among other blocks